source/CardGames.py

```diff
@@ -36,6 +36,14 @@ def setValue(self):
     if self.value > 10:
       self.value = 10
     return self.value
+  
+  def eightCard(self):
+    if self.value == 8:
+      suit = input("What is the new suit? Options: Clubs, Hearts, Diamonds, Spades\n")
+      while suit not in {"Clubs", "Hearts", "Diamonds", "Spades"}:
+        print("Invalid input. Please enter new suit.")
```

Comment:
I think your code is really clean looking which is good! I was trying to do that for my code, but it is something that I need to work on. I think you have a fancy way of doing this task!